src/test/compile-fail/impl-trait/where-allowed.rs

```diff
@@ -125,7 +125,7 @@ trait InTraitDefnReturn {
 // Allowed and disallowed in trait impls
 trait DummyTrait {
     type Out;
```

Comment:
do we have already have a rustfix unit test for this? if not, can we add one?

Reply:
What do you mean by "this"? This particular file? Or the `_: Type` suggestion? I had previously added https://github.com/rust-lang/rust/pull/48309/files#diff-cff76de2568555816771ee1e45d8225f in #48309. Is that what you mean?